### PR TITLE
fix: update th buf cache to get proto deps

### PR DESCRIPTION
**Pull request by gsk967**
## Description

this pr will fix the `make proto-gen` , 

Note: 
**we need to update the commit for deps in buf config whenever  we bump the cosmos-sdk version** 

We can get the commits for deps of buf https://github.com/cosmos/cosmos-sdk/blob/v0.46.2/client/v2/internal/buf.lock

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: proto/buf.yaml [hunk 1]
```diff
@@ -1,10 +1,10 @@
 version: v1
 name: buf.build/umee-network/umee
 deps:
-  - buf.build/cosmos/cosmos-sdk
-  - buf.build/cosmos/cosmos-proto
-  - buf.build/cosmos/gogo-proto
-  - buf.build/googleapis/googleapis
+  - buf.build/cosmos/cosmos-sdk:8cb30a2c4de74dc9bd8d260b1e75e176
```
Comment (robert-zaremba) [inline proto/buf.yaml:4]: Commit `af8...` should be used here.

**Review by robert-zaremba — COMMENTED** [review]
The import overwrite are due to gogo proto version. Let's update gogo proto , and keep the cosmos-sdk commit from umee/main (`af8...`)

**Review by toteki — APPROVED** [review]
Confirmed locally that running `make proto-all` in this branch correcrtly generates the current proto files seen in `main` branch with no diff.

**Comment by robert-zaremba** [comment]
> The import overwrite are due to gogo proto version. Let's update gogo proto , and keep the cosmos-sdk commit from umee/main (`af8...`)

EDIT: ... `af8` is the old one, `8cb30a2c4de74dc9bd8d260b1e75e176` is the correct one

I finished that yesterday in https://github.com/umee-network/umee/pull/1408 but it was skipped in reviews.

**Comment by robert-zaremba** [comment]
Let's merge both - both have same fix, but each has some side improvement.

**Comment by toteki** [comment]
Checking up on the merge conflict - which version to use?

**Comment by gsk967** [comment]
@robert-zaremba  should be we use latest commit for googleapis https://buf.build/googleapis/googleapis/commits/main 
or this commit 8d7204855ec14631a499bd7393ce1970 for googleapis

**Pull request merged**